backend/app/assets/tailwind/application.css

```
@@ -1 +1,40 @@
 @import "tailwindcss";
+
+@theme {
+  /*-------color-------*/
```

Stylelint (17.9.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)